angular/projects/spark-angular/src/lib/directives/sprk-spinner/sprk-spinner.directive.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { SprkSpinnerDirective } from './sprk-spinner.directive';
+
+@Component({
+  selector: 'sprk-test',
+  template: `
+    <div sprkSpinner analyticsString="spinner-1" idString="spinner-1"></div>
+    <div
+      sprkSpinner
+      variant="primary"
+      role="testrole"
+      altText="testvaluetext"
+    ></div>
+    <div sprkSpinner variant="secondary"></div>
+    <div sprkSpinner variant="dark"></div>
+    <div sprkSpinner size="large"></div>
+    <div sprkSpinner additionalClasses="class1"></div>
+  `,
+})
+class TestComponent {}
+
+describe('Spark Spinner Directive', () => {
+  let component: TestComponent;
+  let fixture: ComponentFixture<TestComponent>;
+  let spinner1Element: HTMLElement;
+  let spinner2Element: HTMLElement;
+  let spinner3Element: HTMLElement;
+  let spinner4Element: HTMLElement;
+  let spinner5Element: HTMLElement;
+  let spinner6Element: HTMLElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [SprkSpinnerDirective, TestComponent],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(TestComponent);
+    component = fixture.componentInstance;
+
+    fixture.detectChanges();
+    spinner1Element = fixture.nativeElement.querySelectorAll('div')[0];
+    spinner2Element = fixture.nativeElement.querySelectorAll('div')[1];
+    spinner3Element = fixture.nativeElement.querySelectorAll('div')[2];
+    spinner4Element = fixture.nativeElement.querySelectorAll('div')[3];
+    spinner5Element = fixture.nativeElement.querySelectorAll('div')[4];
+    spinner6Element = fixture.nativeElement.querySelectorAll('div')[5];
+  }));
+
+  it('should create itself', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('add additional classes', () => {
```

Comment:
It's superficial, but let's add the 'should' to these tests for consistency and readability.